app/src/main/java/org/oppia/android/app/profile/AddProfileActivity.kt

```diff
@@ -18,6 +21,10 @@ class AddProfileActivity : InjectableAppCompatActivity() {
     fun createAddProfileActivityIntent(context: Context, colorRgb: Int): Intent {
       val intent = Intent(context, AddProfileActivity::class.java)
       intent.putExtra(ADD_PROFILE_COLOR_RGB_EXTRA_KEY, colorRgb)
+      intent.putProtoExtra(
+        CurrentAppScreenNameIntentDecorator.getCurrentAppScreenNameIntentKey(),
+        CurrentAppScreenNameIntentDecorator.decorateWithScreenName(ScreenName.ADD_PROFILE_ACTIVITY)
```

Comment:
Why not have something like ``currentAppscreenNameDecorator.decorateWithScreenName(ScreenName.ADD_PROFILE_ACTIVITY)``? Then you don't need to expose the key.

You could also use an instance extension function, but they're a bit trickier. Example:

```kotlin
class CurrentAppSCreenNameIntentDecorator {
  fun Intent.decorateWithScreenName(screenName: ScreenName) {...}
}

...

currentAppScreenNameIntentDecorator.run {
  intent.decorateWithScreenName(ScreenName.ADD_PROFILE_ACTIVITY)
}
```

Or, if the extension can go in the decorator's companion object, then you could just import the extension and call it (no ``run`` needed).

Reply:
Thanks! 
I've implemented the extension approach, PTAL :)